app/controllers/report_controller.rb

```diff
@@ -6,6 +6,14 @@ class ReportController < ApplicationController
 
   include ThesisHelper
 
+  def files
```

Comment:
We may want to consider a more explicit name for this. In the future looking at this method it won't be clear that this is really for the `files_with_no_defined_purpose` versus some generic `files` route without having to poke around further.

Reply:
This is fair, but I'd like to move this to a different name once we have an idea of what the alternative report is going to be. Looking at the current reporting epic in Jira, no other requested report seems likely to overlap with this method.